cmd/mesg-cli/orchestrator.go

```diff
@@ -109,9 +94,18 @@ func startOrchestratorCmd(cdc *codec.Codec) *cobra.Command {
 					panic(err)
 				}
 			}()
+
+			// init gRPC server.
+			logger.Info("starting grpc server")
+			server := grpc.New(rpc, ep, orch, viper.GetStringSlice(flagAuthorizedPubKeys))
```

Comment:
had to reverse order of init of the gRPC server as it now requires the orchestrator as dep